handbook/ops/bizops/user_definitions.md

```diff
@@ -46,17 +44,43 @@ Additional Resources:
+| Retained    | A user who were active last month *and* this month                                                                                       |
+| Churned     | A user who were active last month but not this month                                                                                     | 
+| Resurrected | A user who were *not* active last month but are active this month                                                                        |
+| Registered  | A user who created an account on Sourcegraph Cloud                                                                                       |
+
+
+#### Activation qualifying events
+
+Users activate if they perform one of these qualifying events:
+| Product area              | Qualifying events                  | Description                                                                                         |
+|---------------------------|------------------------------------|-----------------------------------------------------------------------------------------------------|
+| Searching for code        | `SearchSubmitted`                  | Submit a search                                                                                     |
+| Navigating the code graph | `findReferences`, `goToDefinition` | Click on go to definition or fin references. Hovering is excluded as it can be performed passively. |
+| Code Insight              | TBD                                | TBD at launch on cloud                                                                              |
+| Batch Changes             | TBD                                | TBD at launch on cloud                                                                              |
+| API docs                  | TBD                                | TBD                                                                                                 |
+
+
+## How are metrics calculated
+Our metrics infrastructure (Looker) gets user counts from our event_logs table (either directly or indirectly). We use the same method as on-premise for calculating user counts, pulling from the Sourcegraph Cloud `event_logs` table. We track unauthenticated users using their `anonymous_user_id`. This is a separate column which contains an anonymous ID, which is stored in a cookie for users that visit Sourcegraph.com. Therefore, for all charts that track Cloud active users, unauthenticated users are included in these counts. 
+
+There are shortcomings to this. For one, when a user converts into an authed user, their events conducted with their anonymous user ID are still in the DB, so we would count two different users being active rather than a single user.For analytics purposes in Amplitude, this is also not ideal because we are not able to connect the actions of a user before and after they've converted.
```

Comment:
```suggestion
There are shortcomings to this. For one, when a user converts into an authed user, their events conducted with their anonymous user ID are still in the DB, so we would count two different users being active rather than a single user. For analytics purposes in Amplitude, this is also not ideal because we are not able to connect the actions of a user before and after they've converted.
```